api/src/test/java/org/openmrs/validator/ConceptValidatorTest.java

```diff
@@ -395,6 +395,9 @@ public void validate_shouldPassForAnEditedConceptWithAMapCreatedWithDeprecatedCo
 	public void validate_shouldNotFailIfATermHasTwoNewMappingsOnIt() {
 		
 		concept.addName(new ConceptName("my name", Context.getLocale()));
+		concept.setConceptClass(new ConceptClass());
```

Comment:
did you have these  tests failing before  these additions

Reply:
Yes @HerbertYiga just an improvement